vendor/cookbooks/rails/recipes/default.rb

```diff
@@ -118,7 +118,9 @@
         domain_names: app_info["domain_names"],
         redirect_domain_names: app_info["redirect_domain_names"],
         enable_ssl: File.exists?("#{applications_root}/#{app}/shared/config/certificate.crt"),
-        custom_configuration: nginx_custom_configuration(app_info))
+        custom_configuration: nginx_custom_configuration(app_info),
+        gzip_enabled: app_info["gzip_enabled"]
```

Comment:
I guess not, since app_info is an application config read from rails settings. There could be more then one application per node config:
```
 35   "active_applications": {
 36     "<appname>_<stage>": {
...
 57       "gzip_enabled": false
 58     }
```